File: pset1/phonebook.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    string name = get_string("What's your name? ");
    int age = get_int("What's your age? ");
    string number = get_string("What's your number? ");

    printf("Age is %s, name is %i, phone is %s\n", name, age, number);
}

File: pset6/lecture/4/phonebook.py
# Saves names and numbers to a csv file

import csv
from cs50 import get_string

# Open a csv file
file = open("phonebook.csv", "a")

# Get name and number
name = get_string("Name: ")
number = get_string("Number: ")

# Print to file
pisaka = csv.writer(file)
pisaka.writerow([name, number])

# Close file
file.close()